src/protocols/secure_channel/CASESession.cpp

```diff
@@ -212,6 +213,7 @@ CHIP_ERROR CASESession::EstablishSession(const Transport::PeerAddress peerAddres
                                          uint16_t localSessionId, ExchangeContext * exchangeCtxt,
                                          SessionEstablishmentDelegate * delegate, Optional<ReliableMessageProtocolConfig> mrpConfig)
 {
+    PW_TRACE_INSTANT("CASESession::EstablishSession", "Commissioning");
```

Comment:
"Commissioning" doesn't sound like the right category. CASE is for operational session establishment.